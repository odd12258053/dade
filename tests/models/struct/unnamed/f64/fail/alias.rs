use dade::model;
#[model]
struct TestModel (
   #[field(alias = "val")]
    f64
);
fn main() {}
