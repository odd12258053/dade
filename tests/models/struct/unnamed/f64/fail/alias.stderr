error: No support alias term on the unnamed field.
 --> tests/models/struct/unnamed/f64/fail/alias.rs:4:4
  |
4 |    #[field(alias = "val")]
  |    ^
